account_mtd/models/mtd_api.py

```diff
@@ -0,0 +1,9 @@
+from odoo import models, fields, api
+
+
+class MtdApiTokens(models.Model):
+    _name = 'mtd.api'
+    _description = "list of api table"
+
+    name = fields.Char('Name', required=True, readonly=True)
```

Comment:
My previous review still stands on all field declarations:
`name = fields.Char()`  is the same as `name = fields.Char('Name')` and `name = fields.Char(string='Name')`.
The only times where a string parameter is required in a field is if the string differs from the fieldname. 

See https://www.odoo.com/documentation/10.0/reference/orm.html#creating-models
```
By default, the field's label (user-visible name) is a capitalized version of the field name, this can be overridden with the string parameter:
```

So I don't think some of these are totally necessary

Reply:
I'm still in two minds about this.  I think @james-opusvl is in the "be explicit" camp, whereas you're saying lean on the automatic code.  I can see benefits to both, but I would say if you're labelling them explicitly, use the kwarg `string='Label'`) rather than leaning on the positional arg order, which varies depending on the type of field.